spec/helpers/givie_helper_spec.rb

```diff
@@ -13,8 +13,9 @@
       homepage: "http://www.foo.bar"
     })
   }
+
   before do
-    ENV['GIVIE_SECRET'] = "kitten sandwich is a strange project name"
+    ENV['GIVIE_SECRET'] = "77a115a459b0499bd8719d5143113fe1"
```

Comment:
Is this a real thing? I would prefer to leave the original value so that we can be sure not to have checked in real tokens